src/PowerShellEditorServices.Protocol/LanguageServer/ShowHelpRequest.cs

```diff
@@ -7,10 +7,10 @@
 
 namespace Microsoft.PowerShell.EditorServices.Protocol.LanguageServer
 {
-    public class ShowOnlineHelpRequest
+    public class ShowHelpRequest
     {
         public static readonly
             RequestType<string, object, object, object> Type =
-            RequestType<string, object, object, object>.Create("powerShell/showOnlineHelp");
+            RequestType<string, object, object, object>.Create("powerShell/showHelp");
```

Comment:
Just to play Devil's advocate here:

Is the reason for the breaking change just that "showOnlineHelp" would be an inaccurate name? Or does possibly getting offline help make the change breaking too?

Just thinking that there are plenty of APIs where the message name doesn't quite mean what it used to because changing the message name is a bigger breaking change than changing what the message does.

It may be that we want to break the message compatibility anyway, but we do have other clients that might benefit from us not changing the name.

Anyway, just want to ask the question: "if we change the functionality but not the message name, is this still a breaking change?" with the followup "if the message name change is the breaking change, is it worth the breakage?"

Reply:
given this is a command in the command pallet and not a settings related change or something along those lines... I think if someone complains we can consider a patch that will re-ADD the old functionality.

Reply:
I think what @rjmholt is talking about is breaking on both sides with the change of the message as PSES isn't used exclusively by vscode-powershell. I'm not opposed to keeping the message name the old one (in fact I think it should be a matter of just backing out a single commit on both sides...)

Reply:
That sounds reasonable!

Reply:
I'm game 👍

Reply:
oh sure, now I need to first remove my big foot from my mouth and figure out how to rewrite the git history...

Reply:
the interactive rebase allowed me to delete the commits that changed the wording of the message (which is partly why I made them separate commits) while still preserving the super difficult minor bug fix commit at the end there.